cmake/developer_package/download/download_and_extract.cmake

```diff
@@ -164,7 +164,13 @@ function (CheckOrDownloadAndExtract component RELATIVE_URL archive_name unpacked
   elseif(USE_NEW_LOCATION)
     set(URL "https://storage.openvinotoolkit.org/dependencies/${RELATIVE_URL}")
   else()
-    set(URL "https://download.01.org/opencv/master/openvinotoolkit/${RELATIVE_URL}")
+    if("${RELATIVE_URL}" STREQUAL "thirdparty/unified/VPU/usb-ma2x8x/firmware_usb-ma2x8x_20221129_35.zip")
```

Comment:
Why can't we reuse USE_NEW_LOCATION for new firmware? 
The current solution looks very rough

Reply:
Just like that https://github.com/openvinotoolkit/openvino/blob/62aac5207909cb13cbddd7fd409b4a63b6393afa/cmake/dependencies.cmake#L108

Reply:
Moreover, [cmake/developer_package/download/download_and_extract.cmake](https://github.com/openvinotoolkit/openvino/pull/14373/files/fa44802bd158b56f87c1bc29019f9e89662cfdff#diff-f50fd4e9576797447d0bae72de4d9244f55845ffb2ff54595b0bf8df3093e00e) should not know anything about VPU. It breaks hierarchy

Reply:
Okay, thanks, will take a look at it now